### PR TITLE
D3ASIM-2708: Adding integration test to validate grid fee working wit…

### PR DESCRIPTION
**Pull request by faizan2590**
…h RedisMarket

### CODE REVIEW

File: integration_tests/test_grid_fee.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+# Connects one client to the house-2 market
+# market = AutoGridFeeUpdateOnMarket('house-2')
+
+
+# Infinite loop in order to leave the client running on the background
+# placing bids and offers on every market cycle.
+# while True:
+#     sleep(0.5)
+#     if market.status == "finished":
+#         break
```
Comment (spyrostz) [inline integration_tests/test_grid_fee.py:56]: Please remove before merging

File: integration_tests/test_grid_fee.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+            self.current_time = market_info['market_info']['start_time']
+            self.updated_fee = self.change_grid_fees_const(self.fee_profile[self.current_time])
+            logging.debug(f"updated_fee: {self.updated_fee}")
+            self.list_dso_stats = self.list_dso_market_stats([self.current_time])
+            assert float(self.list_dso_stats['market_fee_const']) == \
+                   self.fee_profile[self.current_time] == \
+                   float(self.updated_fee['market_fee_const'])
+
+        except AssertionError as e:
+            print(f"Raised exception: {e}. Traceback: {traceback.format_exc()}")
```
Comment (spyrostz) [inline integration_tests/test_grid_fee.py:39]: `logging.error`

**Review by spyrostz — APPROVED** [review]
LGTM

File: integration_tests/test_grid_fee.py (hunk 1 above)
Comment (faizan2590) [reply integration_tests/test_grid_fee.py:56]: Actually left it deliberately for Chris to test it locally to get a feeling regarding how grid fee could be changed.

Comment (spyrostz) [reply integration_tests/test_grid_fee.py:56]: Ah okay then, let it be, thanks for explaining !

**Pull request merged**